std/zig/render.zig

```diff
@@ -717,24 +717,54 @@ fn renderExpression(
                     };
 
                     if (maybe_row_size) |row_size| {
+                        // A place to store the width of each expression and its column's maximum
+                        var widths = try allocator.alloc(usize, exprs.len + row_size);
+                        defer allocator.free(widths);
+                        mem.set(usize, widths, 0);
+
+                        var expr_widths = widths[0 .. widths.len - row_size];
+                        var column_widths = widths[widths.len - row_size ..];
+
+                        // Null stream for counting the printed length of each expression
+                        var null_stream = std.io.NullOutStream.init();
```

Comment:
Minor thing. We have a global `NullOutStream` :)
https://github.com/ziglang/zig/blob/d6455008830eff433a14c5eff6094f2f5ba9991d/std/io.zig#L713